ansible/playbooks/rhel.yml

```diff
@@ -198,55 +201,75 @@
         - ansible_architecture == "S390"
       tags: ccache
 
+    #########
+    #  GIT  #
+    #########
+    - name: Check git version installed
+      shell: git --version | sed -e 's/git version //g' | cut -c 1-3
+      register: git_version
+
     - name: Download git-2.15.0.tar.xz
       get_url:
         url: https://www.kernel.org/pub/software/scm/git/git-2.15.0.tar.xz
         dest: /home/{{ Jenkins_Username }}/git-2.15.0.tar.xz
         mode: 0440
       when:
-        - ansible_distribution_major_version == "7"
-        - ansible_architecture == "ppc64le"
+        - git_version.stdout | version_compare('2.0', '<=', strict=True)
       tags: git-src
 
+    - stat:
+        path=/home/{{ Jenkins_Username }}/git-2.15.0.tar.xz
+      register: git_download
+
     - name: Extract git sources
       unarchive:
         src: /home/{{ Jenkins_Username }}/git-2.15.0.tar.xz
         dest: /home/{{ Jenkins_Username }}
         copy: False
       when:
-        - ansible_distribution_major_version == "7"
-        - ansible_architecture == "ppc64le"
+        - git_download.stat.exists == True 
       tags: git-src
 
     - name: Compile and install git 2.15
-      shell: cd /home/{{ Jenkins_Username }}/git-2.15.0 && make prefix=/usr all && make prefix=/usr install
+      shell: cd /home/{{ Jenkins_Username }}/git-2.15.0 && ./configure --prefix=/usr --without-tcltk && make && make install
```

Comment:
Should be compiling in parallel
cd /home/{{ Jenkins_Username }}/git-2.15.0 && ./configure --prefix=/usr --without-tcltk && ``` make -j {{ ansible_processor_vcpus }} ``` && make install